### DISCUSSION TITLE
关于saas黑盒满分同时的报错信息

### DISCUSSION THREAD

**Discussion by jiqingsong** [physics-data/faq#90] [Q&A] [answered]
程序中最终返回sha256sum时的命令如下
![1630387534(1)](https://user-images.githubusercontent.com/37206109/131446872-a7797933-aba1-424d-b57e-0164aad96fc8.png)
此时黑盒可以满分通过
但同时报错为
![1630387590(1)](https://user-images.githubusercontent.com/37206109/131446950-5c6b3bf5-42b9-4dd1-b018-e829c8a48443.png)
请问原因是什么呢？应该怎么调整呢？

**Comment by Harry-Chen** [comment, marked as answer]
你应该可以忽略这一问题
